pkgs/os-specific/linux/kernel/linux-lqx.nix

```diff
@@ -1,7 +1,7 @@
 { lib, fetchFromGitHub, buildLinux, linux_zen, ... } @ args:
 
 let
-  version = "5.12.14";
+  version = "5.12.19";
   suffix = "lqx2";
```

Comment:
There is `lqx3` as well.

Reply:
Yeah just wanted to get 21.05 in sync with master. We can hopefully just use the backport bot once this is merged.

Reply:
Derp, I didn't notice this was going into `21.05` 🤦.